src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGet.VisualStudio.Contracts.csproj

```diff
@@ -0,0 +1,24 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardVersion)</TargetFramework>
+    <SkipShared>true</SkipShared>
+    <PackProject>true</PackProject>
+    <Shipping>true</Shipping>
+    <IncludeInVsix>true</IncludeInVsix>
+    <IncludeNuGetSharedFiles>true</IncludeNuGetSharedFiles>
+    <RootNamespace>NuGet.VisualStudio.Contracts</RootNamespace>
+  </PropertyGroup>
```

Comment:
How about adding `<Description>one sentence description</Description>` in the csproj file?

Reply:
Added (in a commit yet to be pushed - I really wish GitHub had more conversation 2 states), but `tools-local\doc.tasks` won't compile, so I can't run `build\docs.porj`

Reply:
@dominoFire @nkolev92 Running docs.proj to regenerate the doc is out of scope of this PR, right?

Reply:
I think technically it's @zivkan's responsiblity to do it. 
Don't think it matters in which PR. 

The logic is, you changed it, you update it.

Reply:
https://github.com/NuGet/Home/issues/9721

I'm not going to fix or investigate this further in this PR.

Reply:
> The logic is, you changed it, you update it.

I don't change anything in `tools-local\doc.tasks` :(

Reply:
You added a projecy/changed a project description though :)